unsubscribe-automation/README.md

```diff
@@ -0,0 +1,101 @@
+# Unsubscribe Automation Service
```

Comment:
Move to `apps` folder. `web` is one app. `unsubscribe-automation` is another app.

Reply:
Moved